cosmovisor/README.md

```diff
@@ -281,35 +292,15 @@ mkdir -p $DAEMON_HOME/cosmovisor/genesis/bin
 cp ./build/simd $DAEMON_HOME/cosmovisor/genesis/bin
 ```
 
-For the sake of this demonstration, amend `voting_period` in `genesis.json` to a reduced time of 20 seconds (`20s`):
+Now you can run cosmovisor with simapp v0.44
```

Comment:
This now comes after the user has already attempted to start cosmovisor... Why move the start command and increase the block height earlier in the instructions?

Reply:
because we want to setup the right config at genesis, not when the blockchain started. In multinode setup it would create a failure.